### DISCUSSION TITLE
How to customize the entire theme colors for Dark mode in Chakra UI v3?

### DISCUSSION THREAD

**Discussion by manas-maji** [chakra-ui/chakra-ui#9267] [Q&A] [answered]
Hi,

In Chakra UI v3, the default dark mode colors seem to be heavily reliant on black and related shades. However, I want to globally customize the entire theme's dark mode colors—for example, switching the base dark mode palette to slate or another color scheme that reflects throughout the app (e.g., backgrounds, borders, text, etc.).

Is there a way to define a completely custom color palette for dark mode at the theme level? Ideally, this should affect all components without requiring individual overrides.

Any insights or documentation regarding this would be helpful. If this isn’t supported yet, I believe it could be a valuable feature to enhance theming flexibility.

Thanks in advance for your guidance!

**Comment by segunadebayo** [comment, marked as answer]
Here you go:
https://www.chakra-ui.com/guides/theming-customize-dark-mode-colors

**Reply by manas-maji** [reply]
Thanks a lot!

**Reply by fdcastel** [reply]
After migrating to Chakra UI v3, I found that I preferred the dark theme from v2.

For anyone interested, here's what I'm currently using to replicate the v2 dark theme in v3:

```typescript
semanticTokens: {
  colors: {
    // Source: https://github.com/chakra-ui/chakra-ui/blob/main/packages/react/src/theme/semantic-tokens/colors.ts
    bg: {
      DEFAULT: {
        value: { _light: "{colors.white}", _dark: "#1c202b" /* From Chakra UI v2 */ },
      },
      subtle: {
        value: { _light: "{colors.gray.50}", _dark: "{colors.gray.950}" },
      },
      muted: {
        value: { _light: "{colors.gray.100}", _dark: "{colors.gray.900}" },
      },
      emphasized: {
        value: { _light: "{colors.gray.200}", _dark: "#1a1d28" /* From Chakra UI v2 */ },
      },
      inverted: {
        value: { _light: "{colors.black}", _dark: "{colors.white}" },
      },
      panel: {
        value: { _light: "{colors.white}", _dark: "#303747" /* From Chakra UI v2 */ },
      },
      error: {
        value: { _light: "{colors.red.50}", _dark: "{colors.red.950}" },
      },
      warning: {
        value: { _light: "{colors.orange.50}", _dark: "{colors.orange.950}" },
      },
      success: {
        value: { _light: "{colors.green.50}", _dark: "{colors.green.950}" },
      },
      info: {
        value: { _light: "{colors.blue.50}", _dark: "{colors.blue.950}" },
      },
    },
  },
},
```

This setup was created manually using a color picker, so any suggestions or improvements are welcome!
